packages/authentication/cardstack/middleware.js

```diff
@@ -163,9 +164,28 @@ class Authentication {
     throw new Error(`No such authentication source "${name}"`, { status: 404 });
   }
 
+  async _sessionMeta(ctxt) {
+    let config, ip;
+
+    try {
+       config = await this.searcher.getFromControllingBranch(Session.INTERNAL_PRIVILEGED, 'plugin-configs', '@cardstack/authentication');
```

Comment:
Let's not add a round trip to elasticsearch to every request. We can cache this in the middleware. I'm pretty sure we will already rebuild the middleware stack if a plugin-config is changed.